docs/mixin/dashboards/promscale.json

```diff
@@ -1774,7 +1774,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "exemplar": true,
-          "expr": "1 - promscale_sql_database_health_check_errors_total / promscale_sql_database_health_check_total",
+          "expr": "1 - rate(promscale_sql_database_health_check_errors_total[$__rate_interval] / promscale_sql_database_health_check_total[$health_check_errors_total]",
```

Comment:
Where is `$health_check_errors_total` defined?

Reply:
This looks like a copy/paster error :)